exporter/collector/config.go

```diff
@@ -183,6 +183,11 @@ type LogConfig struct {
 	// ErrorReportingType enables automatically parsing error logs to a json payload containing the
 	// type value for GCP Error Reporting. See https://cloud.google.com/error-reporting/docs/formatting-error-messages#log-text.
 	ErrorReportingType bool `mapstructure:"error_reporting_type"`
+	// InvalidJsonByteStrings exports any Log body of type byte as a raw byte string, when the
+	// byte body does not marshal to valid JSON. Cloud Logging requires byte payloads to marshal to
+	// valid JSON via the encoding/json package. When byte payloads do not translate to valid JSON,
+	// this allows them to still be passed to Cloud Logging as a raw string payload.
+	InvalidJSONByteStrings bool `mapstructure:"invalid_json_byte_strings"`
```

Comment:
Should this be the default behavior?  It seems like by default we should make sure that valid OTLP is delivered to cloud logging.

Reply:
I was thinking users might want to be aware if their valid OTLP isn't valid as a LogEntry. But the collector warning here could be good enough for that. If anything, maybe this option should be inverted (ie, `dont_send_invalid_bytes_json_as_a_string`) but until someone requests that, we can go with this as the default. The expectation seems overwhelmingly to be that if it's valid OTLP, it should end up in cloud logging. Updated